alpha/cpanel-api.yaml

```diff
@@ -76,4 +76,4 @@ resource_types:
   type: docker-image
   source:
     repository: linkyard/concourse-helm-resource
-    tag: 2.9.1
+    tag: latest
```

Comment:
are we sure we want `latest`?

Reply:
They don't do a `2.9` tag to catch all the minor releases. Do we need the helm client to be the same version or lower than Tiller?

Reply:
Changed it to match current server version